GitHubProjectColumns.ps1

```diff
@@ -473,6 +477,13 @@ filter Move-GitHubProjectColumn
         'position' = $Position
     }
 
+    if (-not $PSCmdlet.ShouldProcess($Column, 'Move GitHub Project Column'))
+    {
+        return
+    }
+
+    Write-InvocationLog
```

Comment:
Here's another example (where line 460 won't have an invocation line prior to it)